src/main/java/me/totalfreedom/totalfreedommod/command/Command_settotalvotes.java

```diff
@@ -9,7 +9,7 @@
 import org.bukkit.entity.Player;
```

Comment:
This command should be removed? I can't see why this is ever a thing unless @AtlasMediaGroup/totalfreedom-developers can tell me otherwise?

Reply:
deleted